var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -202,6 +204,25 @@ def configure_args(self):
 
         return args
 
+    def configure(self, spec, prefix):
+        if not self.spec.satisfies("platform=windows"):
+            super(Sqlite, self).configure(spec, prefix)
+
+    def build(self, spec, prefix):
+        if self.spec.satisfies("platform=windows"):
+            nmake = Executable("nmake.exe")
```

Comment:
This is no longer necessary, `nmake` should be set by the build environment setup.